app/scripts/modules/core/pipeline/config/triggers/cron/cronPicker.html

```diff
@@ -41,30 +41,6 @@
           hour<span ng-if="state.hourly.every.hours> 1">s</span>
         </div>
       </div>
-      <div class="row">
-        <div class="col-md-12">
-          <input type="radio"
-                 value="specific"
-                 ng-change="regenerateCron(state)"
-                 ng-model="state.hourly.subTab"
-                 name="hourly-radio">
-          At
-          <select class="form-control input-sm"
-                  ng-change="regenerateCron(state)"
-                  ng-model="state.hourly.specific.hours"
-                  ng-options="hour as padNumber(hour) for hour in selectOptions.hours"
-                  ng-required="state.activeTab === 'hourly' && state.hourly.subTab === 'specific'">
-          </select>
-          :
-          <select class="form-control input-sm"
-                  ng-change="regenerateCron(state)"
-                  ng-model="state.hourly.specific.minutes"
-                  ng-options="minute as padNumber(minute) for minute in selectOptions.minutes"
-                  ng-required="state.activeTab === 'hourly' && state.hourly.subTab === 'specific'">
-          </select>
```

Comment:
It looks like the newer version of this plugin allows users to specify the minutes, e.g. "Every 2 hour(s) on minute 30"; thinking we should do the same